server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -299,19 +320,15 @@ function getNamespaceTables(namespaces) {
     namespaceList = namespaces.toString();
 
     var call = '/rest/tables/namespaces/' + namespaceList;
-    return $.getJSON(call, function(data) {
-      sessionStorage.tables = JSON.stringify(data);
-    });
+    return getJSONForTable(call, 'tables');
```

Comment:
It probably doesn't really matter, but some of these that create the variable, and only use it once inside the function call, could just inline the variable to turn these into one-liners:

```javascript
    return getJSONForTable('/rest/tables/namespaces/' + namespaceList, 'tables');
```

There are several such occurrences in this PR that could be changed like this.